src/charm.py

```diff
@@ -462,6 +480,20 @@ def version(self):
             return info.get("version", None)
         return None
 
+    @property
+    def _external_hostname(self):
+        """Return the external hostname to be passed to ingress via the relation."""
+        # It is recommended to default to `self.app.name` so that the external
+        # hostname will correspond to the deployed application name in the
+        # model, but allow it to be set to something specific via config.
+
+        return self.config["web-external-url"] or f"{self.app.name}.juju"
```

Comment:
```suggestion
        return self.config["web-external-url"] or f"{self.app.name}"
```

Reply:
We sort of stopped doing the `.juju` thing by consensus some time ago... might just be worth grepping the repo to make sure this change doesn't break some docs/tests